### PR TITLE
Add Blogpost: Introducing React Native macOS 0.71

### PR DESCRIPTION
**Pull request by kelset**
Adds blogpost for the announcement of RN macos 0.71.

[Checkout the rendered file here.](https://github.com/microsoft/react-native-devblog/blob/kelset/rnmacos-71-blogpost/blogposts/2023-04-27-announcing-macos-71.md)

### CODE REVIEW

File: blogposts/2023-04-27-announcing-macos-71.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+post_title: "Introducing React Native macOS 0.71"
+author1: sanajmi
+post_slug: 2023-04-27-announcing-macos-71
+post_date: 2023-04-27 12:00:00
```
Comment (kelset) [inline blogposts/2023-04-27-announcing-macos-71.md:5]: this is TBC; waiting for Saad to complete his WP profile.

File: blogposts/2023-04-27-announcing-macos-71.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+### A successful collaboration
+
+A lot of work went into making React Native macOS better for this release, and a special thanks must be given to the Meta and Messenger Desktop engineers that regularly submitted fixes and improvements. In particular, we wanted to shout out [Shawn Dempsey](https://github.com/shwanton) and [Christoph Purrer](https://github.com/christophpurrer) for their great work on the many component (such as TextInput and ScrollView) and adding support for Fabric, among many other things!
+
+### Bringing the new architecture from iOS to macOS
+
+Talking about Fabric: the reason why react-native-macos is a fork and not a fully independent implementation, is that a lot of the existing iOS code can be easily massaged into working for macOS. While this introduces extra complexity in its regular maintenance, it allows to quickly get access to new features -- including the new architecture.
+
+With this release, we are excited to introduce an experimental preview of Fabric for macOS!
```
Comment (kelset) [inline blogposts/2023-04-27-announcing-macos-71.md:29]: note from @stmoy: what are the limitations? Worth adding a mention?

Comment (Saadnajmi) [reply blogposts/2023-04-27-announcing-macos-71.md:29]: There are quite a few limitations. Stuff doesn't render correctly everywhere and I think the app hangs if you click a button 11 times. It is very much not usable, but more a proof of concept.

File: blogposts/2023-04-27-announcing-macos-71.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+To reach this goal, as some of you might have noticed, we had to completely skip some minor releases (such as 0.69 and 0.70). We had to take this tradeoff to accommodate our internal challenges, motivated by the fact that it is the same code that we use in our products (very much like how react-native core is used by Meta). For this reason, it will get more stable as we use it more internally: we push a new patch release on every commit so you can expect in the future new patches to be published as we quickly catch and fix bugs.
+
+### A successful collaboration
+
+A lot of work went into making React Native macOS better for this release, and a special thanks must be given to the Meta and Messenger Desktop engineers that regularly submitted fixes and improvements. In particular, we wanted to shout out [Shawn Dempsey](https://github.com/shwanton) and [Christoph Purrer](https://github.com/christophpurrer) for their great work on the many component (such as TextInput and ScrollView) and adding support for Fabric, among many other things!
```
Comment (Saadnajmi) [inline blogposts/2023-04-27-announcing-macos-71.md:23]:
```suggestion
A lot of work went into making React Native macOS better for this release, and a special thanks must be given to the Meta and Messenger Desktop engineers that regularly submitted fixes and improvements. In particular, we wanted to shout out [Christoph Purrer](https://github.com/christophpurrer), [Shawn Dempsey](https://github.com/shwanton), and [Nick Lefever](https://github.com/lenaic) for adding experimental support for Fabric, among many other things!
```
Changes:
1) Adding Nick Lefever, who did some important work for Fabric
2) Christoph/Shawn weren't the only people who worked on the component fixes, they simply upstreamed the existing fixes they had internally. At the risk of excluding other people, I'd rather we just credit Fabric here which is their unique contribution.

File: blogposts/2023-04-27-announcing-macos-71.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+### Bringing the new architecture from iOS to macOS
+
+Talking about Fabric: the reason why react-native-macos is a fork and not a fully independent implementation, is that a lot of the existing iOS code can be easily massaged into working for macOS. While this introduces extra complexity in its regular maintenance, it allows to quickly get access to new features -- including the new architecture.
```
Comment (Saadnajmi) [inline blogposts/2023-04-27-announcing-macos-71.md:27]:
```suggestion
Talking about Fabric, one of the reasons why React Native macos is a fork and not a fully independent implementation, is that a lot of the existing iOS code can be easily ported into working for macOS. While this introduces extra complexity in its regular maintenance, it allows to quickly get access to new features -- including the new architecture.
```

File: blogposts/2023-04-27-announcing-macos-71.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+This release builds on top of the React Native core - and we encourage you to check out the blogpost highlighting the main changes for the versions between 0.68 and 0.71:
+
+- [Announcing React Native 0.69](https://reactnative.dev/blog/2022/06/21/version-069)
+
+- [Announcing React Native 0.70](https://reactnative.dev/blog/2022/09/05/version-070)
+
+- [React Native 0.71: TypeScript by Default, Flexbox Gap, and more...](https://reactnative.dev/blog/2023/01/12/version-071)
+  - *It's worth noting that the macOS TypeScript types are not fully ready yet - only the core ones are included in this release.*
+
+We have also spent a lot of effort on cleaning up the divergence points between React Native and our fork to make future releases smoother: this includes removing (thousands!) of stale files, upstreaming various bug fixes, and cleaning up RNTester-macOS. In doing so, we were also able to spot bugs and improvements that we sent back upstream, for the benefit of everyone using react-native on any platform.
```
Comment (Saadnajmi) [inline blogposts/2023-04-27-announcing-macos-71.md:44]: > on any platform

It was mostly iOS. But we can keep it like this :P

**Review by Saadnajmi — COMMENTED** [review]
Left my comments, but otherwise looks good! Thanks for all the help :)

File: blogposts/2023-04-27-announcing-macos-71.md (hunk 1 above)
Comment (Saadnajmi) [reply blogposts/2023-04-27-announcing-macos-71.md:5]: Should be good now?

Comment (kelset) [reply blogposts/2023-04-27-announcing-macos-71.md:5]: yes

File: blogposts/2023-04-27-announcing-macos-71.md (hunk 2 above)
Comment (kelset) [reply blogposts/2023-04-27-announcing-macos-71.md]: added an extra line to clarify that it's in proof of concept stage

**Review by Saadnajmi — APPROVED** [review]

**Pull request merged**